opal/mca/btl/usnic/btl_usnic_module.c

```diff
@@ -1613,7 +1613,7 @@ static int create_ep(opal_btl_usnic_module_t* module,
 
     /* Check to ensure that the RX/TX queue lengths are at least as
        long as we asked for */
-    if (channel->info->rx_attr->size < channel->chan_rd_num) {
+    if ((int) channel->info->rx_attr->size < channel->chan_rd_num) {
```

Comment:
This (and the next 2) belong in the prior commit.

Also, we know the RHS will never be negative.  So we should cast the RHS instead of the LHS.